src/Root.tsx

```diff
@@ -1,23 +1,23 @@
-import { FC } from "react";
-import Navigation from "components/Navigation";
+import React, { FC } from "react";
```

Comment:
interesting that we have to import `React` again now, I thought the jsx compiler is still set to `react-jsx`?

Reply:
removed those verbose imports